### PR TITLE
Tanke

### PR DESCRIPTION
**Pull request by elio303**
- Code considers shape of the bug (I don't think this actually works perfectly so can you take a look?)
- Highscore changes according to level selected in start page and when you get a highscore for each level
- Made width of pause button constant.

### CODE REVIEW

File: a2.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
 			   			<p id="timeLeft"></p>
 			    	</div>
 			    	<div>
-				    	<div id="pauseButton" onclick="pauseUnpause()">Pause</div>
+				    	<div id="pauseButton">Pause</div>
```
Comment (wwjzxwjc) [inline a2.html]: dont know about this, i remember the prof prefer just put everything in

Comment (elio303) [inline a2.html]: I read on piazza that we don't need it.

Comment (elio303) [inline a2.html]: Basically all the onclick should be in the javascript page. Not the html page.

File: a2.js [hunk 2]
```diff
@@ -533,34 +608,40 @@ window.onload = function() {
...
-		viewPortContext.arc(x,y-15,3,0,2*Math.PI)
+		viewPortContext.arc(x,y-11,3,0,2*Math.PI)
 		viewPortContext.fillStyle = color;
 		viewPortContext.fill();
 		viewPortContext.strokeStyle = color;
 		viewPortContext.stroke();
 		
 		viewPortContext.beginPath();
-		viewPortContext.arc(x,y-21,3,0,2*Math.PI)
+		viewPortContext.arc(x,y-17,3,0,2*Math.PI)
```
Comment (wwjzxwjc) [inline a2.js]: can i assume these numbers are just done by trial and error? such that they don't exceed 20?

Comment (elio303) [inline a2.js]: Yes

**Pull request merged**